reserved.js

```diff
@@ -327,8 +404,11 @@ async function calculateReserved() {
 
 	// Calculate the reserved balance pallet to pallet
 	reserved = reserved.add(await getDemocracyReserved());
-	reserved = reserved.add(await getElectionsReserved());
+	//reserved = reserved.add(await getElectionsReserved());
```

Comment:
Why is this commented out?

We can't have both, or?

Reply:
I don't think that pallet exists anymore. I think it's been replaced by `phragmenElection`

Reply:
does the program panic though when this is in there?

We should keep it in, and simply return 0 deposit if the pallet doesn't exist

Reply:
No it does not. It simply logs in the console `no elections pallet`.

Sure, I'll add it back

Reply:
Done